packages/nxengine/fix-prefix.patch

```diff
@@ -0,0 +1,19 @@
+diff -uNr a/Makefile.linux b/Makefile.linux
+--- a/Makefile.linux
++++ b/Makefile.linux
+@@ -16,11 +16,11 @@
+ CXX           = g++
+ DEFINES       = # -D_L10N_CP1251
+ CFLAGS        = -pipe -O2 $(DEFINES)
+ CXXFLAGS      = -pipe -O2 $(DEFINES)
+-INCPATH       = -I/usr/include -I/usr/include/SDL -I/usr/local/include -I/usr/local/include/SDL -I.
++INCPATH       = -I$TERMUX_PREFIX/include -I$TERMUX_PREFIX/include/SDL -I$TERMUX_PREFIX/local/include -I$TERMUX_PREFIX/local/include/SDL -I.
+ LINK          = g++
+-LFLAGS        = -Wl,-O1 -Wl,-rpath,/usr/lib
++LFLAGS        = -Wl,-O1 -Wl,-rpath,$TERMUX_PREFIX/lib
+ LIBS_SHARED   = -lSDL -lSDL_ttf
+-LIBS_STATIC   = /usr/local/lib/libSDLmain.a /usr/local/lib/libSDL.a /usr/local/lib/libSDL_ttf.a -ldl -lpthread -lfreetype
++LIBS_STATIC   = $TERMUX_PREFIX/local/lib/libSDLmain.a $TERMUX_PREFIX/local/lib/libSDL.a $TERMUX_PREFIX/local/lib/libSDL_ttf.a -ldl -lpthread -lfreetype
```

Comment:
Why there is a `/local` path component? Remove it. It is not supported officially and shouldn't be used in any of our packages.

If you need it for personal purposes, keep such changes in your fork instead.